src/__test__/navbar.test.js

```diff
@@ -0,0 +1,13 @@
+import { BrowserRouter as Router } from 'react-router-dom';
+import renderer from 'react-test-renderer';
+import Navbar from '../component/Navbar';
+
+it('Renders navbar component', () => {
+  const tree = renderer
+    .create(
+      <Router>
+        <Navbar />
+      </Router>,
+    ).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] For the tests, kindly do the following:

- Write some unit tests for the pure functions written in the redux directory
- Write unit tests/snapshots tests for components like the Home and Detail Components
- Write integration tests using React Testing Library for components. You can read this [article](https://www.digitalocean.com/community/tutorials/how-to-test-a-react-app-with-jest-and-react-testing-library) to have an idea of how to use React Testing Library. 👍

Reply:
Thank you for your feedback. I will make the necessary adjustments and ensure that the code follows the requirements and best practices.

Regarding the broken logo, I will either remove it or replace it with our company's logo. Additionally, I will add an Acknowledgement section to give appropriate credit to Nelson Sakwa as required by the Creative Commons License.

I understand that the mobile design of both the Home and Details page navbar doesn't match the wireframe provided. I will adjust the design to match the given wireframe.

For the Details page, I will set the height of the container to 100vh to hide the white patches caused by the few details on some of the trade companies.

Lastly, I will remove all Eslint disabling comments from the file to ensure that the code adheres to best practices and is easy to read and maintain. Thank you again for your valuable feedback.